src/scss/_settings.scss

```diff
@@ -0,0 +1,92 @@
+html {
+  box-sizing: border-box;
+}
+
+*,
+*::before,
+*::after {
+  box-sizing: inherit;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  margin: 0;
+  background-color: #fff;
+  color: $color-rangoongreen;
+  font: 400 16px / 1.5 Roboto, sans-serif;
+}
+
+body.lock {
+  overflow: hidden;
+}
+
+img {
+  max-width: 100%;
+}
+
+a {
+  text-decoration: none;
+}
+
+.site-container {
+  overflow: hidden; // если используете на сайте position: sticky - уберите эту настройку
```

Comment:
please, clean up code from comments that are not needed; but here you can see examples of [right and bad comments](https://app.works/good-comments-vs-bad-comments/)